tex/stingraypaper.tex

```diff
@@ -296,12 +299,11 @@ \subsection{GX~339--4}
 GX 339--4 is a stellar black hole in a low-mass X-ray binary \citep{Hynesetal03}. 
 The black hole has a lower mass limit of $\sim$\,7\Msun\ \citep{MunozDariasetal08} and possibly a near-maximal spin \citep{Ludlametal15}. 
 The system also likely has a low binary orbit inclination; it has been constrained to $37 \degree <i\lesssim 60\degree$ from optical and X-ray observations \citep{Heidaetal17, Zdziarskietal98}, and spectral modeling by \citet{WangJietal18} estimates $i\approx 40\degree$.
-For this example we use an observation from the  \textit{RXTE} Proportional Counter Array (PCA; \citealt{Jahodaetal96}) in NASA's High Energy Astrophysics Science Archive Research Center (HEASARC) from the 2010 outburst of GX 339--4 \citep{Yamaokaetal10}.
-We are using observation ID 95409-01-15-06 taken from UT 2010-04-22 23:36:52 to UT 2010-04-23 00:01:10.
+We use an observation from the  \textit{RXTE} Proportional Counter Array (PCA; \citealt{Jahodaetal96}) in NASA's High Energy Astrophysics Science Archive Research Center (HEASARC) from the 2010 outburst of GX 339--4 \citep{Yamaokaetal10}, with the observation ID 95409-01-15-06 taken from UT 2010-04-22 23:36:52 to UT 2010-04-23 00:01:10.
 This observation was taken in 64-channel event mode with $122\,\mu$s time resolution (\texttt{E\_125us\_64M\_0\_1s}).
 The following filtering criteria were used to obtain Good Time Intervals (GTIs): Proportional Counter Unit (PCU) 2 is on, two or more PCUs are on, elevation angle $>\,10\degree$, and target offset $<\,0.02\degree$. 
 Time since the South Atlantic Anomaly passage was not filtered on. 
-Applying these filters, we have $\sim$\,1\,ks of good data. 
+Applying these filters, we have $\sim$\,1\,ks of data. 
```

Comment:
I liked _good_ data. That's what filters are for :)

Reply:
Yes, I get it :)
 For me, that makes a _good_ choice of filters but the data is innocent. If you are a calibration expert you might want to see and model those instrumental effects and the data is still _good_ for your purposes.  :^)

Should I delete this change?